configure

```diff
@@ -2144,63 +2154,56 @@ echo "$cxx_version"
 if test "$cxx_verc_fail" = yes ; then
 	echo
 	echo "The version of your compiler is not supported at this time"
-	echo "Please ensure you are using GCC >= 2.95"
```

Comment:
I would add something like "it is too low"

Reply:
Or perhaps "too old"?

Reply:
This check will now fail not because version is bad but because compiler is not recognized.
We support GCC, clang and ICC. If it's not one of them, and not some special SGI compiler, we will end up here.
That's why I removed this sentence.